CMakeLists.txt

```diff
@@ -1,14 +1,24 @@
 cmake_minimum_required(VERSION 3.16)
 
+# API Key for SFMTC api.511.org
+# should be in antioch build dir
+
+# Temporary no key
+file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/build/api_key_511" "FOOBAR")
+
+file(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/build/api_key_511" API_KEY_511)
+
 if(ESP_PLATFORM)
 
-set(EXTRA_COMPONENT_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/espgazer" "${CMAKE_CURRENT_SOURCE_DIR}/fw" "${CMAKE_CURRENT_SOURCE_DIR}/gfx")
+set(EXTRA_COMPONENT_DIRS
+  "${CMAKE_CURRENT_SOURCE_DIR}/espgazer"
+  "${CMAKE_CURRENT_SOURCE_DIR}/fw"
+  "${CMAKE_CURRENT_SOURCE_DIR}/gfx"
+  "${CMAKE_CURRENT_SOURCE_DIR}/esp_components")
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 
-project(antioch)
```

Comment:
don't we need this on regular computer build?

Reply:
Its still there, line 26